addons/test_discuss_full/tests/test_performance.py

```diff
@@ -898,7 +898,7 @@ def _expected_result_for_channel(self, channel):
                 "group_based_subscription": False,
                 "id": channel.id,
                 "invited_member_ids": [["ADD", []]],
-                "is_editable": True,
+                "is_editable": False,
```

Comment:
Isn't this `is_editable` essentially worthless now?
Shouldn't we just check ownership of current user instead?